spec/index.html

```diff
@@ -153,1920 +190,1795 @@ <h2>Semantic Extensions and Entailment Regimes</h2>
+          The more datatypes are <a>recognized</a>, the stronger is the entailment,
+          so that if D &subset; <em>E</em> and <em>S</em> E-entails <em>G</em> then <em>S</em> must <a>D-entail</a> <em>G</em>.
+          Simple entailment is {&nbsp;}-entailment, i.e. D-entailment when D is the empty set,
+          so if <em>S</em> <a>D-entails</a> <em>G</em> then <em>S</em> <a>simply entails</a> <em>G</em>. 
+        </p>
+        <section id="datatype_entailment_patterns" class="informative">
+          <h4>Patterns of datatype entailment (Informative)</h4>
+          <p>Unlike <a>simple entailment</a>, it is not possible to give a single syntactic criterion
+            to detect all D-entailments,
+            which can hold because of particular properties of the lexical-to-value mappings
+            of the <a>recognized</a> datatypes.
+            For example, if D contains <code>xsd:decimal</code> then
+          </p>
+          <p><code>ex:a ex:p "25.0"^^xsd:decimal .</code></p>
+          <p>D-entails</p>
+          <p><code>ex:a ex:p "25"^^xsd:decimal .</code></p>
+          <p>In general, any triple containing a literal with a <a>recognized</a>
+            datatype IRI <a>D-entails</a> another literal when the lexical strings of
+            the literals map to the same value under the lexical-to-value map of the datatype.
+            If two different datatypes in D map lexical strings to a common value,
+            then a triple containing a literal typed with one datatype may <a>D-entail</a>
+            another triple containing a literal typed with a different datatype.
+            For example, <code>"25"^^xsd:integer</code> and <code>"25.0"^^xsd:decimal</code>
+            have the same value, so the above also <a>D-entails</a>
+          </p>
+          <p><code>ex:a ex:p "25"^^xsd:integer .</code></p>
+          <p>when D also contains <code>xsd:integer</code>.</p>
+          <p>(There is a W3C Note [[SWBP-XSCH-DATATYPES]] containing a long
+            <a data-cite="SWBP-XSCH-DATATYPES#sec-values">discussion</a> of literal values.)
+          </p>
+          <p><a>Ill-typed</a> literals are the only way in which a graph can be simply <a>D-unsatisfiable</a>,
+            but datatypes can give rise to a variety of other <a>unsatisfiable</a> graphs
+            when combined with the RDFS vocabulary, defined later.
+          </p>
+        </section>
+      </section>
+    </section>
+    <section id="rdf_d_interpretations">
+      <h2>RDF Interpretations</h2>
+      <p>RDF interpretations impose extra semantic conditions on <code>xsd:string</code>
+        and part of the infinite set of IRIs with the namespace prefix <code>rdf:</code> .
+      <section class="notoc">
+        <h6 class="defcap"><dfn>RDF vocabulary</dfn></h6>
+        <div>
+          <p><code>rdf:type rdf:subject rdf:predicate rdf:object rdf:first rdf:rest rdf:value rdf:nil rdf:List rdf:langString rdf:Property rdf:_1 rdf:_2 ...</code></p>
```

Comment:
It seems odd that this "list" is just a stream, with no punctuation aside from the embedded colons. It would probably be better either vertical or with commas.